src/test/java/StringAddCalculatorTest.java

```diff
@@ -0,0 +1,67 @@
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.NullAndEmptySource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+@DisplayName("문자열 덧셈 계산기")
+public class StringAddCalculatorTest {
+
+    @DisplayName("공백 또는 null 을 입력할 경우 0을 반환한다.")
+    @ParameterizedTest
+    @NullAndEmptySource
+    public void splitAndSum_null_또는_빈문자(String text) {
```

Comment:
```suggestion
    void splitAndSum_null_또는_빈문자(String text) {
```
junit5 부터는 테스트 메서드에서 public 사용은 추천되지 않습니다.
`It is generally recommended to omit the public modifier for test classes, test methods, and lifecycle methods unless there is a technical reason for doing so`

https://junit.org/junit5/docs/current/user-guide/#writing-tests-classes-and-methods 도 참고 부탁드립니다..!

Reply:
네 참고하겠습니다~